tools/license-builder.sh

```diff
@@ -33,7 +33,7 @@ addlicense "c-ares" "deps/cares" \
            "$(sed -e '/^ \*\/$/,$d' -e '/^$/d' -e 's/^[/ ]\* *//' ${rootdir}/deps/cares/src/ares_init.c)"
 addlicense "HTTP Parser" "deps/http_parser" "$(cat deps/http_parser/LICENSE-MIT)"
 addlicense "ICU" "deps/icu" \
-           "$(sed -e '1,/COPYRIGHT AND PERMISSION NOTICE/d' -e '/^<hr/,$d' -e 's/^<\/*p>$//' ${rootdir}/deps/icu/license.html)"
+           "$(sed -e '1,/ICU License - ICU 1\.8\.1 and later/d' -e :a -e 's/<[^>]*>//g;/</N;//ba' ${rootdir}/deps/icu/license.html)"
```

Comment:
No problem. 

!! I had manually formatted the license html into text. How long has this been omitted? Can you stick the license file into the download dir or something so it can be compliant with Icu and icu's dependents?

Icu's license.html I repeat is *going away*. I'll probably name the replacement LICENSE.txt. You could future proof now by checking for such a file and using cat instead.

Feed back welcome here http://bugs.icu-project.org/trac/ticket/12037


Enviado desde nuestro iPhone.